SU-SWS/stanford_basic
# D8CORE-3474: unsetting the white color to fix for a11y

# READY FOR REVIEW

# Summary
- Unset the white color on the search icon/button
- Fixed SOE and SBT

# Review By (Date)
- 3/12

# Urgency
- High 
- A11y violation

# Steps to Test

1. Pull in this change and clear caches.
2. Notice no visual change to the search icon
3. Run Wave.
4. Notice no more error there.

# Affected Projects or Products
- Search button
- SOE
- SBT
- SAA? @katrialesser 

# Associated Issues and/or People
- D8CORE-3474
- @katrialesser I did a preliminary test and there was no visual change to SAA. It should fix it's a11y contrast error too.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
